host/integration_test.go

```diff
@@ -613,7 +613,7 @@ func (s *integrationSuite) TestWorkflowRetry() {
 		RetryPolicy: &workflow.RetryPolicy{
 			InitialIntervalInSeconds:    common.Int32Ptr(int32(initialIntervalInSeconds)),
 			MaximumAttempts:             common.Int32Ptr(int32(maximumAttempts)),
-			MaximumIntervalInSeconds:    common.Int32Ptr(1),
+			MaximumIntervalInSeconds:    common.Int32Ptr(10),
```

Comment:
Let's change this back to 1 to make sure the maximum interval is enforced.

Reply:
Make sense. Updated